custom_components/tuya_local/devices/djive_arc_fan.yaml

```diff
@@ -0,0 +1,101 @@
+name: djive Flowmate ARC Humidifier
+products:
+  - id: bf5fd02f13d3f6d92exy2t
+    name: djive ARC Humidifier
+primary_entity:
+  entity: fan
+  name: Fan
```

Comment:
Yeah I did that because the name had humidifier in it, but the primary entity is indeed not a humidifier, but a fan. But if I use a more generic name like you suggested below, I think this should solve it as well.